website/src/theme/BlogLayout/index.js

```diff
@@ -1,14 +1,18 @@
 import React from "react";
 import { useLocation } from "@docusaurus/router";
 import ThemeLayout from "@theme/Layout";
+import useBaseUrl from "@docusaurus/useBaseUrl";
 
 import { Layout } from "../../components";
 
 export default function BlogLayout({ children, ...props }) {
   const location = useLocation();
+  const blogUrl = useBaseUrl("blog");
 
   const isHomeOrTagsPage =
-    location.pathname === "/blog" || location.pathname.includes("/blog/tags");
+    location.pathname === blogUrl ||
+    location.pathname === blogUrl + `/` ||
+    location.pathname.includes("/blog/tags");
```

Comment:
do we need to change this too?
```suggestion
    location.pathname.includes(blogUrl + '/tags');
```

Reply:
It works as is, thanks to `.includes`, but it doesn't hurt so I'm applying it. Or we can replace it with `.startsWith`.